app/models/game.rb

```diff
@@ -236,4 +226,36 @@ def en_passant_opportunity_active?
     @en_passant_capture_opportunity
   end
 
+  private
+
+    def populate_board!
+      # white pieces
+      create_pieces 'white', 1, 2
+      # black pieces
+      create_pieces 'black', 8, 7
+    end
+    
+    def create_pieces(color, row, pawn_row)
+      # Enumerator of Piece types in the correct order for chess
+      non_pawn = [Rook, Knight, Bishop, Queen, King, Bishop, Knight, Rook].to_enum
+      # Creates each Piece type in 'non_pawn' plus a pawn
+      non_pawn.with_index(1).each do |piece, i|
+        if color.eql? 'white'
+          piece.create(game_id: id, x_coordinate: i, y_coordinate: row, color: color, player_id: white_player_id)
+          Pawn.create(game_id: id, x_coordinate: i, y_coordinate: pawn_row, color: color, player_id: white_player_id)
+        else
+          piece.create(game_id: id, x_coordinate: i, y_coordinate: row, color: color)
+          Pawn.create(game_id: id, x_coordinate: i, y_coordinate: pawn_row, color: color)
+        end
+      end
+    end
+    
+    def set_first_turn!
+      update_attribute(:turn, white_player_id)
+    end
+    
+    def assign_black_pieces!
+      pieces.where(color: 'black').each { |piece| piece.update_attribute(:player_id, black_player_id) }
```

Comment:
You could of done this in same place you assigned white_player pieces. Although I'm not sure why you would need to.